### PR TITLE
Update intro.qmd

### PR DESCRIPTION
**Pull request by SarahanneMField**

### CODE REVIEW

File: intro.qmd [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 This is a book created from markdown and executable code.
 
+I like big butts and I cannot lie.
```
Comment (chartgerink) [inline intro.qmd:5]: That is inappropriate.

File: intro.qmd [hunk 2]
```diff
@@ -2,4 +2,6 @@
...
+
```
Comment (chartgerink) [inline intro.qmd:6]:
```suggestion
I like big peaches and I cannot lie.

```

File: intro.qmd (hunk 1 above)
Comment (dvanravenzwaaij) [inline intro.qmd:5]: Yeah, not cool

**Review by chartgerink — CHANGES_REQUESTED** [review]
This is of course just a test. Tanks.

**Pull request merged**